app/components/log-drain-actions/template.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="panel-heading-actions">
+  {{#unless logDrain.isDeprovisioning}}
+    {{#unless logDrain.isProvisioning}}
+      <button class="btn btn-default btn-xs" {{action "restart" logDrain}}>Restart</button>
+    {{/unless}}
+    <button class="btn btn-default btn-xs" {{action "deprovision" logDrain}}>Deprovision</button>
+  {{/unless}}
```

Comment:
@gib -- Let's move this logic into computed properties.  Nested `#unless` are a bit harder to follow than something like `{{#if showRestartButton}}`.  Computeds are also easier to unit test.